packages/editor/src/components/parts/program/activity/ProgramInterfaceStartPart.tsx

```diff
@@ -32,7 +32,9 @@ const ProgramInterfaceStartPart = () => {
 
   return (
     <>
-      <JavaClassSelector javaClass={config.javaClass} onChange={change => update('javaClass', change)} type='ACTIVITY' />
+      {(thirdParty == undefined || thirdParty == false) && (

```

Comment:
first why only two '==' ? in js you should always use three...
And second, you can also use `thirdParty!!` instead

Reply:
thanks, but if I set thirdParty!! I get the following error:
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/7dc8a375-a076-43e3-8337-5e9e6f779f10)

Reply:
or `!!thirdParty`

Reply:
ah you check for falsy, yes of course then it is just `!thirdParty`